plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java

```diff
@@ -35,4 +60,14 @@ public static String toYamlString(CNode rootNode) throws IOException {
         serializeYamlNode(yamlRoot, buffer);
         return buffer.toString();
     }
+
+    public static String toStringFromYamlFile(Object clazz, String resourcePath) throws URISyntaxException, IOException {
```

Comment:
We should add some java docs to all of these? To all of misc?

Reply:
done